### PR TITLE
feat: add test for viewRecordWrapper

### PR DESCRIPTION
**Pull request by muenzpraeger**
### What does this PR do?
Adds tests for the `viewRecordWrapper` components

### What issues does this PR fix or reference?
none

### The PR fulfills these requirements:
[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### CODE REVIEW

File: force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+describe('c-view-record-wrapper', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+    });
+
+    it('passes recordId to c-example-wrapper component c-view-record component', () => {
```
Comment (pozil) [inline force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js:14]: Fix test name: "passes recordId to c-example-wrapper component"

File: force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        // Create initial element
+        const element = createElement('c-view-record-wrapper', {
+            is: ViewRecordWrapper
+        });
+        element.recordId = RECORD_ID;
+        document.body.appendChild(element);
+
+        // Verify c-view-record component exists, and that recordId is passed to its attribute
+        const viewRecordEl = element.shadowRoot.querySelector(
+            'c-example-wrapper > c-view-record'
```
Comment (pozil) [inline force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js:38]: Simplify selector to just `c-view-record`

File: force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js (hunk 1 above)
Comment (muenzpraeger) [reply force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js]: Implemented.

File: force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js (hunk 2 above)
Comment (muenzpraeger) [reply force-app/main/default/lwc/viewRecordWrapper/__tests__/viewRecordWrapper.test.js]: Implemented.

**Pull request merged**